.gitignore

```diff
@@ -15,18 +14,7 @@ local.properties
 .signing/
  
 # User-specific configurations
-.idea/libraries/
-.idea/workspace.xml
-.idea/tasks.xml
-.idea/.name
-.idea/compiler.xml
-.idea/copyright/profiles_settings.xml
-.idea/encodings.xml
-.idea/inspectionProfiles/Project_Default.xml
-.idea/misc.xml
-.idea/modules.xml
-.idea/scopes/scope_settings.xml
-.idea/vcs.xml
+.idea/
```

Comment:
There's no need to specify a whole pile of stuff here when we're ignoring the whole folder anyway